### PR TITLE
Upgrade to 1.41.1

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

**Comment by Lctrs** [comment]
Ping @amtlib-dot-dll

File: build.py [hunk 1]
```diff
@@ -201,15 +201,16 @@ def get_python2_recipe():
...
-    subprocess.run('curl https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-' + python_version + '.tar.bz2 | sudo tar -xjf - --directory /', shell=True, check=True)
-
-    packages = ['autopep8', 'pylint', 'pipenv', 'ipython', 'rope', 'flake8', 'yapf']
-    patterns = {
-        '.whl': re.compile(r'(?P<package>.*)-(?P<version>.*?)-.*?-.*?-.*?\.whl'),
-        '.tar.gz': re.compile(r'(?P<package>.*)-(?P<version>.*?)\.tar\.gz'),
-        '.zip': re.compile(r'(?P<package>.*)-(?P<version>.*?)\.zip')
-    }
     with tempfile.TemporaryDirectory() as tmpdir:
+        subprocess.run('curl https://storage.googleapis.com/travis-ci-language-archives/python/binaries/ubuntu/16.04/x86_64/python-' + python_version + '.tar.bz2 | tar -xjf - --directory ' + tmpdir, shell=True, check=True)
```
Comment (amtlib-dot-dll) [inline build.py]: This line was for running inside Travis CI to download a specific version of Python, in order to generate the JSON recipe by Travis.  The downloaded tarball is not used after this change

File: build.py [hunk 2]
```diff
@@ -594,9 +601,6 @@ def parse_repo(base_recipe):
                             'type': 'archive',
                             **get_url_sha512('https://nodejs.org/dist/v' + nodejs_version + '/node-v' + nodejs_version + '.tar.xz')
                         }
-                    ],
```
Comment (amtlib-dot-dll) [inline build.py]: Why are these lines removed?

Comment (amtlib-dot-dll) [reply build.py]: @Lctrs I've sent you an invitation to my repo using Travis to run `build.py`, by the way thank you for your contribution for this project!
